internal/cmd/measurement_schema/client.go

```diff
@@ -0,0 +1,241 @@
+		req = req.Org(params.OrgName)
+	} else if !params.OrgID.Valid() && params.OrgName == "" {
+		req = req.Org(c.CLI.ActiveConfig.Org)
+	}
+
+	if params.BucketID.Valid() {
+		req = c.BucketApi.GetBuckets(ctx).Id(params.BucketID.String())
+	} else if params.BucketName != "" {
+		req = c.BucketApi.GetBuckets(ctx).Name(params.BucketName)
+	} else {
+		return nil, errors.New("must specify bucket ID or bucket name")
+	}
+
+	resp, _, err := c.BucketApi.GetBucketsExecute(req)
+	if err != nil {
+		return nil, fmt.Errorf("failed to find bucket %q: %w", params.BucketName, err)
+	}
+	buckets := resp.GetBuckets()
+	if len(buckets) == 0 {
+		return nil, fmt.Errorf("no bucket found with name %q", params.BucketName)
+	}
+
+	return &orgBucketID{OrgID: buckets[0].GetOrgID(), BucketID: buckets[0].GetId()}, nil
+}
+
+type CreateParams struct {
+	internal.OrgBucketParams
+	Name           string
+	Columns        []api.MeasurementSchemaColumn
+	ExtendedOutput bool
+}
+
+func (c Client) Create(ctx context.Context, params CreateParams) error {
+	ids, err := c.resolveOrgBucket(ctx, params.OrgBucketParams)
+	if err != nil {
+		return err
+	}
+
+	res, _, err := c.BucketSchemasApi.
+		CreateMeasurementSchema(ctx, ids.BucketID).
+		OrgID(ids.OrgID).
+		MeasurementSchemaCreateRequest(api.MeasurementSchemaCreateRequest{
+			Name:    params.Name,
+			Columns: params.Columns,
+		}).
+		Execute()
+
+	if err != nil {
+		return err
+	}
+
+	return c.printMeasurements(ids.BucketID, []api.MeasurementSchema{res}, params.ExtendedOutput)
+}
+
+type UpdateParams struct {
+	internal.OrgBucketParams
+	Name           string
+	ID             string
+	Columns        []api.MeasurementSchemaColumn
+	ExtendedOutput bool
+}
+
+func (c Client) Update(ctx context.Context, params UpdateParams) error {
+	ids, err := c.resolveOrgBucket(ctx, params.OrgBucketParams)
+	if err != nil {
+		return err
+	}
+
+	var id string
+	if params.ID == "" && params.Name == "" {
+		return errors.New("measurement id or name required")
+	} else if params.ID != "" {
+		id = params.ID
+	} else {
+		id, err = c.resolveMeasurement(ctx, *ids, params.Name)
+		if err != nil {
+			return err
+		}
+	}
+
+	res, _, err := c.BucketSchemasApi.
+		UpdateMeasurementSchema(ctx, ids.BucketID, id).
+		OrgID(ids.OrgID).
+		MeasurementSchemaUpdateRequest(api.MeasurementSchemaUpdateRequest{
+			Columns: params.Columns,
+		}).
+		Execute()
+
+	if err != nil {
+		return err
+	}
+
+	return c.printMeasurements(ids.BucketID, []api.MeasurementSchema{res}, params.ExtendedOutput)
+}
+
+type ListParams struct {
+	internal.OrgBucketParams
+	Name           string
+	ExtendedOutput bool
+}
+
+func (c Client) List(ctx context.Context, params ListParams) error {
+	ids, err := c.resolveOrgBucket(ctx, params.OrgBucketParams)
+	if err != nil {
+		return err
+	}
+
+	req := c.BucketSchemasApi.
+		GetMeasurementSchemas(ctx, ids.BucketID).
+		OrgID(ids.OrgID)
+
+	if params.Name != "" {
+		req = req.Name(params.Name)
+	}
+
+	res, _, err := req.Execute()
```

Comment:
I like this style of calling `Execute()` on the request, will make a refactor ticket to switch existing files over to it.